actor-typed/src/main/scala/org/apache/pekko/actor/typed/scaladsl/Routers.scala

```diff
@@ -185,5 +185,5 @@ trait PoolRouter[T] extends Behavior[T] {
   /**
    * Any message that the predicate returns true for will be broadcast to all routees.
    */
-  def withBroadcastPredicate(pred: T => Boolean): PoolRouter[T]
+  def withBroadcastPredicate(predicate: T => Boolean): PoolRouter[T]
```

Comment:
Pre @He-Pin , this change may not source safe, because scala [Named Arguments](https://docs.scala-lang.org/tour/named-arguments.html)